dashboard/app/controllers/videos_controller.rb

```diff
@@ -44,6 +44,12 @@ def create
     filename = upload_to_s3
     @video = Video.new(video_params.merge(download: "https://videos.code.org/#{filename}"))
 
+    if @video.locale != 'en-US'
+      unless Video.exists?(key: @video.key, locale: 'en-US')
```

Comment:
nit; can we use something like `i18n.default_locale`, both here and in the model?

Reply:
Done!